configure.ac

```diff
@@ -2031,43 +2032,43 @@
 
     AM_CONDITIONAL([HAVE_LUA], [test "x$enable_lua" != "xno"])
 
-  # libgeoip
+  # libmaxminddb
```

Comment:
I see libmaxminddb and libmaxminddb-devel RPM packages for CentOS 6 available on the web.

I don't see falling back on libgeoip as a viable option going forward because the database file format that library uses has been discontinued by MaxMind.  The DB cannot be downloaded anymore from their site.  So a new Suricata install could not use the geoip rule option with the legacy database and the libgeoip library unless the user has his own private old copy of the database.

Reply:
OK, if old but supported distribution have it I don't see any reason but the existing build system to use the old geoip.